src/e2e/java/uk/gov/hmcts/reform/sscs/functional/IssueFurtherEvidenceHandlerFunctionalTest.java

```diff
@@ -48,32 +48,33 @@ private void verifyEvidenceIsNotIssued() {
         SscsCaseData caseData = caseDetails.getData();
         assertEquals("failedSendingFurtherEvidence", caseData.getHmctsDwpState());
         List<SscsDocument> docs = caseData.getSscsDocument();
-        assertNull(docs.get(0).getValue().getEvidenceIssued());
-        assertEquals("No", docs.get(1).getValue().getEvidenceIssued());
-        assertEquals("No", docs.get(2).getValue().getEvidenceIssued());
-        assertEquals("No", docs.get(3).getValue().getEvidenceIssued());
+        Assertions.assertThat(docs)
+            .extracting(SscsDocument::getValue)
+            .extracting(SscsDocumentDetails::getEvidenceIssued)
+            .contains("No", "No", "No");
     }
 
     private void verifyEvidenceIssued() {
         SscsCaseDetails caseDetails = findCaseById(ccdCaseId);
         SscsCaseData caseData = caseDetails.getData();
         List<SscsDocument> docs = caseData.getSscsDocument();
 
-        assertNull(docs.get(0).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(1).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(2).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(3).getValue().getEvidenceIssued());
+        Assertions.assertThat(docs)
+            .extracting(SscsDocument::getValue)
+            .extracting(SscsDocumentDetails::getEvidenceIssued)
+            .contains("Yes", "Yes", "Yes");
     }
 
     private void verifyEvidenceIssuedAndReasonableAdjustmentRaised() {
         SscsCaseDetails caseDetails = findCaseById(ccdCaseId);
         SscsCaseData caseData = caseDetails.getData();
         List<SscsDocument> docs = caseData.getSscsDocument();
 
-        assertNull(docs.get(0).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(1).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(2).getValue().getEvidenceIssued());
-        assertEquals("Yes", docs.get(3).getValue().getEvidenceIssued());
+        Assertions.assertThat(docs)
+            .extracting(SscsDocument::getValue)
+            .extracting(SscsDocumentDetails::getEvidenceIssued)
+            .contains("Yes", "Yes", "Yes");
```

Comment:
.contains("Yes", "Yes", "Yes")
can be simplied/made more elegant by doing:
.hasSize(3)
.containsOnly("Yes")

Reply:
I'll make a call here and say that while Lucas' suggestion at first glance appears to be marginally better in this particular context, it's still subjective and I'm happy to keep the current implementation.

I'll mark this conversation as resolved.